torch/_inductor/select_algorithm.py

```diff
@@ -197,6 +197,11 @@ def store_output(self, indices, val, mask):
                 V.graph.sizevars.simplify(s) for s in self.output_node.get_size()
             ]
             assert len(indices) == len(lengths)
+            replacements = {
+                x: self.args.size(x)
+                for x in index_symbols + lengths
+                if x.name.startswith("s") or x.name.startswith("ps")
```

Comment:
Curious if there is a more robust way of checking this. Or do the `s*` and `ps*` necessarily cover everything we want to have in the `replacements`? As written, I feel it's easy to miss some cases in the future. Maybe there is a helper function (or define one if not)?

Reply:
I copied it from https://github.com/pytorch/pytorch/blob/main/torch/_inductor/codegen/common.py#L928 haha. Let me extract it into a function.